### ISSUE TITLE
Release Ci once again failed to build windows executables

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#5285]
... and this time I only noticed after sending the release announcement *sigh*.

This is essentially a variant of #5011

The failed job: https://github.com/gap-system/gap/actions/runs/3722840754/jobs/6314090227

Immediate cause reported at https://github.com/gap-packages/agt/issues/15

This is really annoying, and hints at multiple problems:

1. of course I should have noticed the missing windows binaries
2. but since humans miss such things,  *our automation* should have noticed -- e.g. the scripts updating the website could have bailed out and refused the output based on the absence of those files
3. the CI tests for the PackageDistro perhaps could have also picked this one up, by checking for broken symlinks (or perhaps really *any* kind of symlinks, as those seem to be a repeated source of issues on Windows)
4. we lack a good mechanism to "heal" such issues: after all, it is trivial to work around the issue (just delete the offending symlink). But in practice I don't see any good way to achieve this without release 4.12.3: I can't just insert this `rm PATH` invocation into the workflow. And I also can't just re-tag, as it quite likely would produce new tarballs with differing shasums.

How to resolve this now? I see these options:
1. release GAP 4.12.3 with *identical* source code, just a `rm -f pkg/agt/doc/mathjax` added (urgh, doesn't seem appealing)
2. find a way to inject that `rm` into a re-run of the CI job (I see no way to do that, though I guess if we had `tmate` integration set up for that job, perhaps that would allow for it...)
3. I could re-tag 4.12.2 after all, after first downloading all relevant tarballs; then after all CI run, replace any of the new tarballs that changed SHA256
4. someone (@ChrisJefferson perhaps) could perhaps build the GAP .exe "manually" and upload it to the release

**Comment by fingolfin** [comment]
Ok one more idea: I can try to hack the release CI script in my fork of this repository and create a 4.12.2 tag there to produce the desired .exe files.

**Comment by fingolfin** [comment]
Also, another minor action to take against this kind of issue: let's add a step to dev/releases/README.md that suggest to check that all files are present.

another could be to have a "check release" script or GH workflow that checks this (and also verifies shasums while at it)

**Comment by ChrisJefferson** [comment]
I have added windows 4.12.2 to the windows release, obviously, that doesn't fix the underlying problem, but it solves the short term issue!

**Comment by fingolfin** [comment]
@ChrisJefferson thank you. That's only 64 bit .exe files, right? That's fine by me, though I guess it should have been announced that we drop 32bit Windows support.... Ah well, at this point, I say we just wait and see if someone complains

**Comment by ChrisJefferson** [comment]
Yes, I don't think we can even build 32-bit any more, without digging out an old copy of the 32-bit cygwin setup.exe, they replaced the one on the website with an executable that just says "nope, no 32-bit any more".

**Comment by fingolfin** [comment]
The "spiritual successor" to ReleaseTools is https://github.com/gap-actions/release-pkg/ and it indeed checks for symlinks and other problems.

All in all, we didn't have a similar issue in the past 3 years, and I don't see anything immediately actionable left in this issue beyond what we've already done. In the end, no matter what we do, there is always a chance something goes wrong; but overall I feel we are in a good position these days.

So I am closing this issue now.
